### PR TITLE
ADAPT-4519 | @rebeccahongsf | create form page-type component

### PR DESCRIPTION
**Pull request by rebeccahongsf**
# READY FOR REVIEW

# Summary
- create form page-type component
**Note:** Started some styling in this PR. Additional styling and review with design to come within ADAPT-4416

# Review By (Date)
- when possible :D

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Review the following pages 
  - [`/travel-study/additional-payment-form`](https://deploy-preview-309--stanford-alumni.netlify.app/travel-study/additional-payment-form/)
  - [`/travel-study/italy-notify-me-form`](https://deploy-preview-309--stanford-alumni.netlify.app/travel-study/italy-notify-me-form/)
  - [`/travel-study/custom-journey-form`](https://deploy-preview-309--stanford-alumni.netlify.app/travel-study/custom-journeys-form)
3. Review [StoryBlok form page-type](https://app.storyblok.com/#!/me/spaces/145938/stories/0/0/115412127)
- Confirm that site editor can add either one `WYSIWYG` or `TripNotifyMe` block under Form Information
- Confirm that site editor can add one `GiveGabForm` block under GiveGab Form
- Confirm that if a `TripNotifyMe` block is used, the tab title is source from the `TripNotifyMe` block, not the `Page Title (H1)` field on the form page.
4. Review code and confirm that the necessary fields for the form page is here. 

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

# Associated Issues and/or People
- ADAPT-4519

### CODE REVIEW

File: src/components/page-types/formPage/formPage.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import { Container } from '../../layout/Container';
+import { Heading } from '../../simple/Heading';
+import Layout from '../../partials/layout';
+import CreateBloks from '../../../utilities/createBloks';
+import getNumBloks from '../../../utilities/getNumBloks';
+import Ankle from '../../partials/ankle/ankle';
+import { HeroImage } from '../../composite/HeroImage/HeroImage';
+import { Grid } from '../../layout/Grid';
+import { GridCell } from '../../layout/GridCell';
+// import AuthenticatedPage from '../../auth/AuthenticatedPage';
```
Comment (sherakama) [inline src/components/page-types/formPage/formPage.js:12]: Is this commented out for testing? Can we not enable this?

File: src/tailwind/plugins/theme/colors.js [hunk 2]
```diff
@@ -22,9 +22,17 @@ module.exports = function () {
       DEFAULT: '#181D1C',
       dark: '#070B0A',
       opacity: {
+        40: 'rgba(24, 29, 28, 0.4)',
         80: 'rgba(24, 29, 28, 0.8)',
       },
     },
+    'saa-black-new': {
```
Comment (sherakama) [inline src/tailwind/plugins/theme/colors.js:29]: Are you sure you want to name this `-new`?

Comment (sherakama) [reply src/tailwind/plugins/theme/colors.js:29]: New get old fast.

File: src/components/page-types/formPage/tripNotifyMe.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  return (
+    <SbEditable content={blok}>
+      {/* <SAACtaLink
+        leadingIcon="arrow-left"
+        size="default"
+        link={tripURL}
+        className="su-rs-mb-6"
+      >
+        Back to {tripTitle}
+      </SAACtaLink> */}
```
Comment (sherakama) [inline src/components/page-types/formPage/tripNotifyMe.js:59]: Remove commented-out code.

File: src/components/page-types/formPage/tripNotifyMe.js [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+  const tripDuration = useMemo(() => {
+    const { days: dayDuration } = getDuration(startDate, endDate);
+
+    if (dayDuration >= 0) {
+      const days = dayDuration + 1;
+      return `${days} day${days === 1 ? '' : 's'}`;
+    }
+
+    return '';
+  }, [startDate, endDate]);
```
Comment (sherakama) [inline src/components/page-types/formPage/tripNotifyMe.js:48]: Luxon is available and installed into this project and could make this easier: https://moment.github.io/luxon/#/math?id=diffs

File: src/components/page-types/formPage/tripNotifyMe.js [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+    <SbEditable content={blok}>
+      {/* <SAACtaLink
+        leadingIcon="arrow-left"
+        size="default"
+        link={tripURL}
+        className="su-rs-mb-6"
+      >
+        Back to {tripTitle}
+      </SAACtaLink> */}
+      <Helmet titleTemplate={title} title={title} />
```
Comment (sherakama) [inline src/components/page-types/formPage/tripNotifyMe.js:60]: This works well 👍

**Review by sherakama — CHANGES_REQUESTED** [review]
This is looking good. Just a couple of comments to address.

File: src/components/page-types/formPage/tripNotifyMe.js (hunk 4 above)
Comment (sherakama) [reply src/components/page-types/formPage/tripNotifyMe.js:48]: Ah, getDuration is luxon and it doesn't automatically do the plural for you? Darn. Ok forget this comment.

File: src/components/page-types/formPage/formPage.js (hunk 1 above)
Comment (rebeccahongsf) [reply src/components/page-types/formPage/formPage.js:12]: This has been commented out for testing. Would it be easier to leave it comment out for now until Elena has finished styling the GG forms or should we just provide `zguan`'s login credentials? 😃

File: src/tailwind/plugins/theme/colors.js (hunk 2 above)
Comment (rebeccahongsf) [reply src/tailwind/plugins/theme/colors.js:29]: Ack! Great question and point — I'll check in with Bhavika and rename the new black variant :)

Comment (rebeccahongsf) [reply src/tailwind/plugins/theme/colors.js]: Looks like the saa-black-new is the same as the saa-black. Reverted the colors file and added the opacity 40 option  👍

File: src/components/page-types/formPage/formPage.js (hunk 1 above)
Comment (sherakama) [reply src/components/page-types/formPage/formPage.js:12]: Elena has test user credentials but sometimes has issues logging in. We've identified it is a problem with firefox and will ask that she try a different browser when it happens.

Comment (sherakama) [reply src/components/page-types/formPage/formPage.js:12]: Please enable so we get the 'full' experience.

**Review by sherakama — APPROVED** [review]
GTG when you enable the auth component.

**Pull request merged**